build_vars.bzl

```diff
@@ -1,3 +1,3 @@
-BUILD_SDK_VERSION = 34
-BUILD_TOOLS_VERSION = "32.0.0"
+BUILD_SDK_VERSION = 35
+BUILD_TOOLS_VERSION = "34.0.0"
```

Comment:
Do we need to make any documentation changes to account for these?

Reply:
A description of these changes already exist in the wiki. While there are some wiki changes needed on account of Gradle being removed, they don't block this PR.